README.rst

```diff
@@ -165,6 +165,16 @@ When the switch ``enabled`` is set to ``true``, HTTP Basic Authentication is ena
 
 If HTTP Basic Authentication is enabled then the CLI will send HTTP requests using HTTPS instead of HTTP.
 
+bndl
+^^^^
+
+The ``bndl`` command is used to create or modify bundles. It can be used for the following:
+
+- Creating a bundle from Docker and OCI images
```

Comment:
The shazar entry (right below this) already used `-` and it seems to render okay. See https://github.com/typesafehub/conductr-cli/pull/435/files#diff-88b99bb28683bd5b7e3a204826ead112R183